cli_tools/common/utils/compute/metadata_gce.go

```diff
@@ -12,12 +12,14 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-//+build !test
+//go:build !test
+// +build !test
```

Comment:
Did the presubmit tests fail and require you to change this? If not, I'd recommend reverting it.

If it's not blocking this PR, but you think it's a good change, I'd recommend doing a separate PR since this change appears (to me) to be unrelated to this PR.

Reply:
Let's check :) I've dropped this commit.

I had a presubmit failure before, because the gofmt tool suggested some changes, therefore I just called it locally and made a commit. But I believe you're right here, I don't remember this diff in my presubmit failure 🤔

Reply:
It works, thank you!